packages/manager/apps/vrack-services/src/components/layout-helpers/OnboardingLayout.component.tsx

```diff
@@ -93,39 +97,86 @@ export const OnboardingLayout: React.FC<OnboardingLayoutProps> = ({
           {description}
         </OsdsText>
         <div className="flex mb-9 sm:py-8 xs:w-full xs:flex-col xs:space-y-4 sm:items-center sm:w-fit sm:flex-row sm:space-x-4 sm:space-y-0">
-          {primaryButtonLabel && (primaryHref || primaryOnClick) && (
-            <OsdsButton
-              inline
-              disabled={primaryButtonDisabled || undefined}
-              color={ODS_THEME_COLOR_INTENT.primary}
-              size={primaryButtonSize}
-              href={primaryHref}
-              {...(primaryOnClick ? handleClick(primaryOnClick) : {})}
-            >
-              {primaryButtonLabel}
-            </OsdsButton>
-          )}
-          {secondaryButtonLabel && (secondaryHref || secondaryOnClick) && (
-            <OsdsButton
-              inline
-              disabled={secondaryButtonDisabled || undefined}
-              color={ODS_THEME_COLOR_INTENT.primary}
-              variant={ODS_BUTTON_VARIANT.stroked}
-              size={secondaryButtonSize}
-              href={secondaryHref}
-              target={secondaryTarget || OdsHTMLAnchorElementTarget._blank}
-              {...(secondaryOnClick ? handleClick(secondaryOnClick) : {})}
-            >
-              {secondaryButtonLabel}
-              <span slot={secondaryButtonIconPosition}>
-                <OsdsIcon
-                  name={secondaryButtonIcon}
-                  size={ODS_ICON_SIZE.xs}
-                  color={ODS_THEME_COLOR_INTENT.primary}
-                />
-              </span>
-            </OsdsButton>
-          )}
+          {primaryButtonLabel &&
+            !(iamActions || urn) &&
```

Comment:
To have a more concise code we can either do :
```
const Button = !iamActions || !urn ? OsdsButton : ManagerButton;

return (
<Button inline disabled={...
```

or store the props in an object and pass them to the right component :
```
const primaryButtonProps = {
  inline: true,
  disabled: primaryButtonDisabled || undefined,
  ...
}

!iamActions || !urn ? <OsdsButton {...primaryButtonProps} /> : <ManagerButton {...primaryButtonProps} />
```